tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -834,33 +849,33 @@ def test_batch_drift(self):
             function_name="batch-drift-function",
             context=context,
             infer_results_df=infer_results_df,
-            trigger_monitoring_job=True,
+            # trigger_monitoring_job=True,  # TODO: comment out when ML-5792 is done
```

Comment:
It is a deprecated argument why to comment out? and why you haven't delete it?

Reply:
Done in fbbf7b0